stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/model/AnnotatedClass.java

```diff
@@ -66,7 +66,7 @@
 
         FieldOption fieldOption = useStag != null ? useStag.value() : null;
         if (fieldOption == null) {
-            useStag = findParentUseStagAnnotation(element);
+            useStag = ElementUtils.findAnnotation(UseStag.class, element);
```

Comment:
Understood, this change is in keeping with the discussion on the related issue 👍